torch_xla/csrc/BUILD

```diff
@@ -0,0 +1,289 @@
+        "@com_google_absl//absl/hash",
+        "@com_google_absl//absl/memory",
+        "@com_google_absl//absl/strings",
+        "@com_google_absl//absl/types:optional",
+        "@com_google_absl//absl/types:span",
+        "@org_tensorflow//tensorflow/compiler/tf2xla/kernels:conv_op_helpers",
+        "@org_tensorflow//tensorflow/compiler/xla:comparison_util",
+        "@org_tensorflow//tensorflow/compiler/xla:literal_util",
+        "@org_tensorflow//tensorflow/compiler/xla:permutation_util",
+        "@org_tensorflow//tensorflow/compiler/xla:shape_util",
+        "@org_tensorflow//tensorflow/compiler/xla:types",
+        "@org_tensorflow//tensorflow/compiler/xla/client:xla_builder",
+        "@org_tensorflow//tensorflow/compiler/xla/client/lib:arithmetic",
+        "@org_tensorflow//tensorflow/compiler/xla/client/lib:comparators",
+        "@org_tensorflow//tensorflow/compiler/xla/client/lib:constants",
+        "@org_tensorflow//tensorflow/compiler/xla/client/lib:logdet",
+        "@org_tensorflow//tensorflow/compiler/xla/client/lib:math",
+        "@org_tensorflow//tensorflow/compiler/xla/client/lib:slicing",
+        "@org_tensorflow//tensorflow/compiler/xla/stream_executor:dnn",
+        "@org_tensorflow//tensorflow/core:lib",
+        "@org_tensorflow//tensorflow/core/framework:tensor_shape",
+        "@org_tensorflow//tensorflow/core/kernels:conv_ops",
+        "@org_tensorflow//tensorflow/core/lib/core:errors",
+        "@org_tensorflow//tensorflow/core/lib/gtl:inlined_vector",
+        "@org_tensorflow//tensorflow/core/profiler/lib:traceme",
+        "@org_tensorflow//third_party/python_runtime:headers",
+        "@pybind11//:pybind11_embed",  # libpyton
+    ],
+)
+
+ptxla_cc_library(
+    name = "device",
+    srcs = ["device.cpp"],
+    hdrs = ["device.h"],
+    deps = [
+        "//third_party/xla_client:computation_client",
+        "//third_party/xla_client:util",
+        "@com_google_absl//absl/strings",
+        "@com_google_absl//absl/types:optional",
+        "@org_tensorflow//tensorflow/compiler/xla/client:xla_builder",
+    ],
+)
+
+ptxla_cc_library(
+    name = "computation",
+    srcs = ["computation.cpp"],
+    hdrs = ["computation.h"],
+    deps = [
+        "//third_party/xla_client:computation_client",
+        "@org_tensorflow//tensorflow/compiler/xla/client:xla_builder",
+        "@org_tensorflow//tensorflow/compiler/xla/client:xla_computation",
+    ],
+)
+
+ptxla_cc_library(
+    name = "layout_manager",
+    srcs = ["layout_manager.cpp"],
+    hdrs = ["layout_manager.h"],
+    deps = [
+        ":device",
+        "//third_party/xla_client:debug_macros",
+        "//third_party/xla_client:sys_util",
+        "//third_party/xla_client:tf_logging",
+        "//third_party/xla_client:util",
+        "@com_google_absl//absl/strings",
+        "@org_tensorflow//tensorflow/compiler/xla:shape_util",
+        "@org_tensorflow//tensorflow/core/framework:tensor_shape",
```

Comment:
same question, do we need this `tensorflow/core` deps directly or generally?

we would migrate to pull XLA from OpenXLA, and would delete tensorflow too, so would delete `tensorflow/core` deps too